ide/coqide.opam

```diff
@@ -6,14 +6,19 @@ bug-reports: "https://github.com/coq/coq/issues"
 dev-repo: "https://github.com/coq/coq.git"
 license: "LGPL-2.1"
 
-available: [ocaml-version >= "4.02.3"]
+available: [ocaml-version >= "4.05.0"]
 
 depends: [
-  "dune"      { build }
-  "ocamlfind" { build }
+  "dune"               { build & >= "1.2.0" }
+  "ocamlfind"          { build }
   "num"
-  "camlp5"
+  "camlp5"             {         >= "7.02"  }
+  "conf-gtksourceview"
+  "lablgtk"            {         >= "2.18.5" }
```

Comment:
This is inconsistent with what is documented in `INSTALL`.

Reply:
Fixed, see new changes.